app/views/producer_mailer/order_cycle_report.html.haml

```diff
@@ -37,10 +38,11 @@
         %tr
           %td
             #{line_items.first.variant.sku}
+          - if @distributors_pickup_times.many?
+            %td
+              #{line_items.first.product.supplier.name}
```

Comment:
I think we can use `=` instead of interpolation here, and below ;)

Reply:
Nice pick, I cleaned up the file